orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -60,6 +60,9 @@ export PGSSLMODE=verify-full
 {%- endif %}
 
 errorExit() {
+  if [[ "$CLOSECONNECTIONS" == "true" ]]; then
```

Comment:
I agree. It will be helpful.

Reply:
Hi @RokolyA94 , I added the service type log in the limit_incomming_connection() function because whenever this function is called it will doLog. The test screenshot (the "after" one) has been updated with this too. Plz let me know if this would work for you!Thank you!